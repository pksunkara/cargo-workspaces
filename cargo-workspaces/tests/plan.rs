mod utils;
use insta::assert_snapshot;

#[test]
fn test_plan_single() {
    // `err` may contain a warning about missing `http.cainfo` config value.
    let (out, _err) = utils::run("../fixtures/single", &["ws", "plan"]);
    assert_snapshot!(out);
}

#[test]
fn test_plan_normal() {
    // `err` may contain a warning about missing `http.cainfo` config value.
    let (out, _err) = utils::run("../fixtures/normal", &["ws", "plan"]);
    assert_snapshot!(out);
}

#[test]
fn test_plan_normal_long() {
    // `err` may contain a warning about missing `http.cainfo` config value.
    let (out, _err) = utils::run("../fixtures/normal", &["ws", "plan", "--long"]);
    assert_snapshot!(out);
}
