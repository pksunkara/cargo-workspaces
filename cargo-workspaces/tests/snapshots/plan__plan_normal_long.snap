---
source: tests/plan.rs
expression: out
---
dep1 v0.1.0 dep1
dep2 v0.1.0 dep2
top  v0.1.0 top

